src/nft/components/collection/Activity.css.ts

```diff
@@ -112,6 +112,23 @@ export const addressCell = style([
   },
 ])
 
+globalStyle(`${addressCell} a[href]`, {
+  color: `${themeVars.colors.textPrimary}`,
+  textDecoration: 'none',
+})
+
+globalStyle(`${addressCell} a[href]:hover`, {
+  color: `${themeVars.colors.textPrimary}`,
+  opacity: `${themeVars.opacity.hover}`,
+  textDecoration: 'none',
+})
+
+globalStyle(`${addressCell} a[href]:focus`, {
```

Comment:
instead of global style can you just use a styled a link component (using styled-components)?